circuits/circuits/send.circom

```diff
@@ -12,8 +12,10 @@ template Send(levels) {
     signal output new_root;
 
     // make sure it's a send request by checking sender and recipient is non-zero
-    IsZero(in <== sender) === 0;
-    IsZero(in <== recipient) === 0;
+    signal is_sender_zero = IsZero(in <== sender);
+    is_sender_zero === 0;
+    signal is_recipient_zero = IsZero(in <== recipient);
+    is_recipient_zero === 0;
```

Comment:
I realise the original code was written by me, but I'm pretty sure `IsZero(in <== send)` is invalid syntax 😅
Don't we have to do:
```
    component is_sender_zero = IsZero();
    is_sender_zero.in <==  sender;
    is_sender_zero.out === 0;
```

Reply:
this was introduced in 2.1: https://docs.circom.io/circom-language/anonymous-components-and-tuples/

Reply:
and yeah the `=` is wrong here. should be `<==`. was planning to update that but you caught me before.